### ISSUE TITLE
Authentication failed

### ISSUE THREAD

**Issue by mspyerz** [streamlink/streamlink-twitch-gui#869]
### Checklist

- [X] I understand that **Streamlink Twitch GUI** is just a launcher for **Streamlink**
- [X] [I have read the contribution guidelines](https://github.com/streamlink/streamlink-twitch-gui/blob/master/CONTRIBUTING.md)
- [X] [I have checked the list of open and recently closed issues](https://github.com/streamlink/streamlink-twitch-gui/issues?q=is%3Aissue+sort%3Aupdated-desc)
- [X] [I have checked the commit log of the master branch](https://github.com/streamlink/streamlink-twitch-gui/commits/master)

### Streamlink Twitch GUI version

v1.11.0

### Streamlink version

v3.1.1

### Operating system, environment and configuration details

windows 10 21H2

### Description

Welp decided to update my streamlink twitch gui and that was my first mistake. Now i'm stuck on the Authentication failed whenever i try to sign on. Ive downloaded everything according to the wiki: node.js, git and yarn. I rebuild it using the git code. I do all these jumps and still auth failed. What am i missing? Did read about deleting the config file but couldn't find it in %localdata% or w/e. I even went into my twitch and cancelled the connection. I tried to redo it and it looked like it was doing something but failed ultimately. This isn't a bug but it's bugging me. I'll even take a step-by-step video as a response. I love the app and enjoy the customization.

**Comment by quietvoid** [comment]
Getting the same issue when trying to log in.
The token request seems to result in a 500 error: `Error: Adapter operation failed`.

**Comment by bastimeyer** [comment]
> v1.11.0

You'll have to upgrade. Read the changelog.
https://github.com/streamlink/streamlink-twitch-gui/releases/tag/v2.0.0
https://github.com/streamlink/streamlink-twitch-gui/issues/849

> The AUR packaging situation seems to be a bit messy.

No, just read the PKGBUILD. The AUR package has been moved to `streamlink-twitch-gui-bin` and the old one will be upgraded after I've initiated the package merge request, which I will do soon. Again, read the PKGBUILD.
